docs/vfolder/vfolder.rst

```diff
@@ -141,29 +141,36 @@ Delete folder
 -------------
 
 If you have permission to delete the storage folder, you can delete it by
-clicking the trash can icon in the Control panel. When you click the Delete
-button, a confirm dialog appears. To prevent accidental deletion, you have to
-enter the name of the folder to be deleted, explicitly.
+clicking the 'trash can' icon in the control column. By clicking the icon, 
```

Comment:
```suggestion
clicking the 'trash can' button in the control column. By clicking the icon, 
```

Reply:
https://github.com/lablup/backend.ai-docs-webui/pull/64#issuecomment-2149091268 Same in here.